### ISSUE TITLE
Extend the Plugin API to allow clients to easily disable selected plugins

### ISSUE THREAD

**Issue by phantomjinx** [hawtio/hawtio-next#706]
The hawtio-online Discover plugin would like to be able to disable the connect plugin. At the moment, the following workaround has to be executed:
```
 for (const plugin of hawtio.getPlugins()) {
    if (plugin.id === 'connect') {
      log.debug('Disabling connect plugin')
      plugin.isActive = async () => {
        return false
      }
    }
  }
```
see: https://github.com/hawtio/hawtio-online/blob/main/packages/online-shell/src/bootstrap.tsx#L17-L24

This is a bit of a hack and an easier API could be provided.

reference: https://github.com/hawtio/hawtio-online/pull/244#pullrequestreview-1772235968

**Comment by tadayosi** [comment]
@phantomjinx Probably this #788 fixes the issue as well. So, with the fix now you can select what plugins to use for hawtio online:
```
// Bootstrap Hawtio
//registerPlugins()

// You can also select which builtin plugins to load
connect()
jmx()
camel()
```
Do you think we can close this with the fix?

**Comment by phantomjinx** [comment]
Fixed by #788.
